src/sentry/static/sentry/app/components/platformList.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import {Flex} from 'grid-emotion';
+
+import Platformicon from 'app/components/platformicon';
+
+const MAX_PLATFORMS = 3;
```

Comment:
This would be better as a prop since I could see some instances where the parent will want to control the number of platforms to display.